src/dms/frontend/EdFi.DataManagementService.Frontend.BulkLoader/EdFi.DataManagementService.Frontend.BulkLoader.csproj

```diff
@@ -0,0 +1,57 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
The term "Bulk Loader" is already used by another Ed-Fi product. Naming this the same thing will cause great confusion. Call this` EdFi.DataManagementService.Frontend.SchoolYearLoader`